### ISSUE TITLE
Request option: shibmd:scope:0:unknownonly

### ISSUE THREAD

**Issue by cdbesten** [OpenConext/OpenConext-engineblock#1290]
When you have an IdP where shibmd:scope can not be applied, we want to limit schacHomeOrganization as much a possible.

The request would be to create a reverse filter on the value of schacHomeOrganization. Only allow a value which is not scoped in any of the other connected IdPs.

**Comment by thijskh** [comment]
There's a feature in Engineblock that you can set the field `coin:schachomeorganization` for an IdP _A_, say to `example.gov`. This does (obviously, as it's Engineblock...) _two_ things:

* It sets the value of the released schacHomeOrganization of this IdP statically to `example.gov` on outgoing assertions, regardless of the input attribute value.
* It makes this value reserved, i.e. other IdPs cannot deliver this schacHomeOrganization anymore, it's checked that incoming SchacHomeOrganization of an IdP _B_ is not listed in `coin:schachomeorganization` for any other IdP.

We do not use this feature currently, so have no real world evidence of doing this. It's just what the feature's supposed to do. Does this feature help you with your use case?

**Comment by tvdijen** [comment]
Nope, IdP's can release multiple home-orgs, and that's where we want to use the shibmd:scope feature.
The issue with that is that it validates _more_ than just the homeOrg-attribute (and i.e. our principalName doesn't match the scope).

**Comment by baszoetekouw** [comment]
@tvdijen I think the feature you're requesting here is orthogonal to @cdbesten's question.  I think the feature to make the attributes which are checked against `shib:scope` configurable, sounds useful.

@cdbesten is your usecase covered by the `coin:schachomeorganisation` feature?  Or do you need to check the scope of other attributes apart from `schacHomeOrganisation`?

**Comment by cdbesten** [comment]
@baszoetekouw No. Our usecase would be to prevent a 'trusted proxy' to use any of the sHOs defined in the other connected IdPs. 

This would be a security measure to prevent 'trusted proxies' to proxy account logins from sHOs would should never be proxies through them since there is a direct connection.
